Dockerfile

```diff
@@ -28,7 +28,12 @@ RUN adduser \
     --uid "${UID}" \
     "${USER}"
 
+# Speed up build by leveraging docker layer caching
+COPY go.mod go.sum ./
+RUN go mod download
```

Comment:
Do we really need this? We removed it because of the vendoring.

Reply:
yeah, this shouldn't be needed any more.